InductivePredicates.dfy(64,10): Error: assertion violation
Execution trace:
    (0,0): anon0
    (0,0): anon3_Then
InductivePredicates.dfy(76,11): Error: assertion violation
Execution trace:
    (0,0): anon0

Dafny program verifier finished with 29 verified, 2 errors
